src/main/java/frc/team2412/robot/Hardware.java

```diff
@@ -38,7 +38,7 @@
         private static final Mk4SwerveModuleHelper.GearRatio GEAR_RATIO;
 
         static {
-            GEAR_RATIO = Robot.getInstance().isCompetition()
+            GEAR_RATIO = Robot.instance.isCompetition()
                     ? Mk4SwerveModuleHelper.GearRatio.L2
                     : Mk4SwerveModuleHelper.GearRatio.L1;
```

Comment:
This really ought to be a static method that takes the robot type as a parameter & returns a `GearRatio`.